<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoursePage</class>
 <widget class="QDialog" name="CoursePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:light gray</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcomelabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>WELCOME TO THE COURSE SELECTION PAGE</string>
   </property>
  </widget>
  <widget class="QLabel" name="courselabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Course : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="falltermbutton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>161</width>
     <height>100</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="text">
    <string>Fall Term</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adminmode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Admin Mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="courseCompleted">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>161</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mark Course as Complete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wintertermbutton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>161</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Winter Term</string>
   </property>
  </widget>
  <widget class="QLabel" name="courseselectedbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>761</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background_color:black</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="facultylabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Faculty : </string>
   </property>
  </widget>
  <widget class="QComboBox" name="CoursesComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>591</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="activeUserLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User Active:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="facultyComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>591</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="adminerrormessage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="goToTimetable">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>161</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Timetable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goToTimetable_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Progress</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearWinterTerm">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>240</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Winter Term</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearFallTerm">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Fall Term</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goToTimetable_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Notifications</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
